hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerMetrics.java

```diff
@@ -45,14 +45,14 @@
   private MutableCounterLong movedContainersNumInLatestIteration;
 
   @Metric(about = "Number of iterations that Container Balancer has run for.")
-  private MutableCounterLong countIterations;
+  private MutableCounterLong numIterations;
 
   @Metric(about = "Number of datanodes that were involved in balancing in the" +
       " latest iteration.")
   private MutableCounterLong datanodesNumInvolvedInLatestIteration;
 
   @Metric(about = "Amount of data in Gigabytes that is causing unbalance.")
-  private MutableCounterLong dataSizeCausingUnbalanceGB;
+  private MutableCounterLong dataSizeUnbalancedGB;
 
   @Metric(about = "Number of unbalanced datanodes.")
   private MutableCounterLong datanodesNumUnbalanced;
```

Comment:
I will also recommend changing names of other metrics. Like datanodesNum -> numDatanodes and movedContainersNum -> numMovedContainers.
We can do it in a separate PR as well.

Reply:
Oh, I remember we had decided on this naming convention in a previous PR: https://github.com/apache/ozone/pull/2230#discussion_r637872787

But this does conflict with `numIterations` where "num" is the prefix.

Reply:
I see. My idea is that it should be easy to search these metrics in prometheus. Maybe we need to have common prefix for balancer then. Can we check some naming guide here?

Reply:
I looked into some examples. We've used "num" as a prefix in other places, such as `numCacheHits`, `numCacheMisses`, `numInFlightReplications`, `numWriteStateMachineOps` etc.

Reply:
@lokeshj1703 I've fixed the naming.